### PR TITLE
fix: fix error on uapp generator scaffolding tools

### PR DESCRIPTION
**Pull request by darsene**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-11509
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- ~Only FR translations have been updated~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~Breaking change is mentioned in relevant commits~

### CODE REVIEW

File: packages/manager/tools/sao-ovh-manager-app/template/src/app.module.js [hunk 1]
```diff
@@ -128,9 +128,9 @@ export default async (containerEl, shellClient) => {
                     : null,
                 },
               },
-              { location: false }
-            )
-          }
+              { location: false },
+            );
+          }rm
```
Comment (anooparveti) [inline packages/manager/tools/sao-ovh-manager-app/template/src/app.module.js:133]: **nitpick:** unwanted change no?
```suggestion
          }
```

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/tools/sao-ovh-manager-app/template/src/app.module.js (hunk 1 above)
Comment (darsene) [reply packages/manager/tools/sao-ovh-manager-app/template/src/app.module.js:133]: Totally ! Good catch!

Comment (darsene) [reply packages/manager/tools/sao-ovh-manager-app/template/src/app.module.js]: It's fixed

**Review by anooparveti — APPROVED** [review]

**Review by cbourgois — APPROVED** [review]

**Comment by antleblanc** [comment]
**question:** Shouldn't commit type being a `fix` rather than `build`?

**Review by antleblanc — APPROVED** [review]

**Pull request merged**